ts/utils/keychain.ts

```diff
@@ -17,7 +17,9 @@ const PIN_KEY = "PIN";
  * Saves the provided PIN in the Keychain
  */
 export async function setPin(pin: PinString): Promise<boolean> {
-  return await Keychain.setGenericPassword(PIN_KEY, pin);
+  return await Keychain.setGenericPassword(PIN_KEY, pin, {
```

Comment:
Can you create a wrapper of `setGenericPassword` here in `utils` that adds the `accessible: Keychain.SecAccessible.WHEN_UNLOCKED_THIS_DEVICE_ONLY` option, and use that in both places? Also link to the relevant documentation describing `WHEN_UNLOCKED_THIS_DEVICE_ONLY`